assets/js/atomic/blocks/product/button/index.js

```diff
@@ -13,7 +13,7 @@ import { ProductButton } from '@woocommerce/atomic-components/product';
 import sharedConfig from '../shared-config';
 
 const blockConfig = {
-	title: __( 'Product Button', 'woo-gutenberg-products-block' ),
+	title: __( 'Add to Cart Button', 'woo-gutenberg-products-block' ),
```

Comment:
While I agree with this change and like it, this changes the localized string, thus breaking any existing translations that might exist for this string.  Are we okay with that?  Maybe check to see how well this is translated?

Reply:
I think this one and the summary were the most important as the current strings are not very descriptive. Leaving these in place.